build_tools/python/e2e_test_framework/models/matmul.py

```diff
@@ -108,45 +108,42 @@
     input_types=["123x2561xf32", "2561x2561xf32", "123x2561xf32"],
 )
 
-# TODO(#17344): regenerate .mlirbc files
-# MATMUL_1x256x2048_I8xI4_MLIR = common_definitions.Model(
-#     id=unique_ids.MICRO_MATMUL_1x256x2048_I8xI4_MLIR,
-#     name="matmul_1x256x2048_i8_i4_i32_tile_config_default",
-#     tags=["i8i4", "ubench", "matmul"],
-#     source_type=common_definitions.ModelSourceType.EXPORTED_LINALG_MLIR,
-#     source_url="https://storage.googleapis.com/iree-model-artifacts/linalg/jax_derived_0.4.25_1710887127/matmul_1x256x2048_i8_i4_i32_tile_config_default.mlirbc",
-#     entry_function="main",
-#     input_types=["1x256xi8"],
-# )
+MATMUL_1x256x2048_I8xI4_MLIR = common_definitions.Model(
+    id=unique_ids.MICRO_MATMUL_1x256x2048_I8xI4_MLIR,
+    name="matmul_1x256x2048_i8_i4_i32_tile_config_default",
+    tags=["i8i4", "ubench", "matmul"],
+    source_type=common_definitions.ModelSourceType.EXPORTED_LINALG_MLIR,
+    source_url="https://storage.googleapis.com/iree-model-artifacts/jax/jax_models_0.4.30_1719457850/DOT_PRODUCT_JAX_1X256X2048XI8I4/linalg.mlirbc",
+    entry_function="main",
+    input_types=["1x256xi8"],
+)
 
 MATMUL_256x256x2048_I8xI4_MLIR = common_definitions.Model(
     id=unique_ids.MICRO_MATMUL_256x256x2048_I8xI4_MLIR,
     name="matmul_256x256x2048_i8_i4_i32_tile_config_default",
     tags=["i8i4", "ubench", "matmul"],
     source_type=common_definitions.ModelSourceType.EXPORTED_LINALG_MLIR,
-    source_url="https://storage.googleapis.com/iree-model-artifacts/linalg/jax_derived_0.4.25_1710887127/matmul_256x256x2048_i8_i4_i32_tile_config_default.mlirbc",
+    source_url="https://storage.googleapis.com/iree-model-artifacts/jax/jax_models_0.4.30_1719457850/DOT_PRODUCT_JAX_256X256X2048XI8I4/linalg.mlirbc",
     entry_function="main",
     input_types=["256x256xi8"],
 )
 
-# TODO(#17344): regenerate .mlirbc files
```

Comment:
Thanks (especially for the PR description with steps followed)!

Tagging https://github.com/iree-org/iree/issues/17344